snippets/fer1_annoted.js

```diff
@@ -40,11 +40,13 @@ clustal.read(url, function(err, seqs) {
 xhr("./data/fer1.gff3", function(err, request, body) {
   var features = gffParser.parseSeqs(body);
   m.seqs.addFeatures(features);
+  m.render();
```

Comment:
Unrelated to my changes: not rendering after adding features causes some weird visual glitch.